pynet/datasets/hcp.py

```diff
@@ -0,0 +1,104 @@
+import os
+from collections import namedtuple
+
+import boto3
+import nibabel as nib
+import numpy as np
+import pandas as pd
+
+Item = namedtuple("Item", ["input_path", "output_path", "metadata_path"])
+
+
+def nii_to_np_arr(filename, low=False, binary=False):
+    img = nib.load(filename)
+    img_data = img.get_data()
+    img_data = np.append(img_data[2:-2, :, 2:-2], np.zeros((256, 9, 256)), axis=1)
+    if low:
+        img_data = img_data.reshape(32, 8, 40, 8, 32, 8).mean(axis=(1, 3, 5))
+
+    if binary:
+        img_data = 1 * (img_data > img_data.mean())
```

Comment:
Strange. If you want to binarize the array you can do arr[arr > 0] = 1.

Reply:
arr[arr > 0] = 1 makes the mask bigger in low resolution. That's why I used the mean. Using scipy.ndimage.zoom, values can also become greater than 1 or less than 0